pkg/artifactory/resource_artifactory_remote_helm_repository.go

```diff
@@ -36,8 +35,8 @@ var helmRemoteSchema = mergeSchema(baseRemoteSchema, map[string]*schema.Schema{
 type HelmRemoteRepo struct {
 	RemoteRepositoryBaseParams
 	HelmChartsBaseURL            string   `hcl:"helm_charts_base_url" json:"chartsBaseUrl,omitempty"`
```

Comment:
I know it isn't part of your changes, but I think `omitempty` should be removed from here too. @alexhung what do you think?

Reply:
I agree we should remove this too, assuming tests all passed. If test(s) fails, then let's open a separate issue to for this. I don't want scope creep.

Reply:
Removed it! Could one of you run the tests? I have only been testing manually by modifying `sample.tf`.

Reply:
As the field is required and cannot be `""`, I do not think `omitempty` has any effect here.

Reply:
@tenstad The commit passed our internal CI so this is good to go.